pkg/helm/environment/environment.go

```diff
@@ -46,13 +46,16 @@ type EnvSettings struct {
 	Debug bool
 	// KubeContext is the name of the kubeconfig context.
 	KubeContext string
+	// KubeConfig is the name of the kubeconfig file.
+	KubeConfig string
 }
 
 // AddFlags binds flags to the given flagset.
 func (s *EnvSettings) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar((*string)(&s.Home), "home", DefaultHelmHome, "location of your Helm config. Overrides $HELM_HOME")
 	fs.StringVar(&s.TillerHost, "host", "", "address of Tiller. Overrides $HELM_HOST")
 	fs.StringVar(&s.KubeContext, "kube-context", "", "name of the kubeconfig context to use")
+	fs.StringVar(&s.KubeConfig, "kubeconfig", "", "path to kubeconfig file")
```

Comment:
kubeconfig should override KUBECONFIG as do the other global flags e.g. $TILLER_NAMESPACE overrides `--tiller-namespace`, `--host` overrides $HELM_HOST etc.

Reply:
so we want the $KUBECONFIG env variable to override the kubeconfig cli?

Reply:
no, but we want to read the default as what's in $KUBECONFIG like we do with $HELM_HOST et al.

Reply:
Would you mind updating this doc string to say "path to kubeconfig file. Overrides $KUBECONFIG" to maintain consistency with the rest of the flags?

Reply:
Looks like this doc string still needs to be updated as per @bacongobbler's comment

Reply:
Done